### DISCUSSION TITLE
Deroghe ad assenze

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#433] [Richieste di aiuto]
Buonasera, volevo sapere se fosse possibile porre in deroga il conteggio di determinate assenze dal conteggio finale.
Ho la necessità di togliere dal conteggio di un'alunna alcune assenze, che non devono essere conteggiate ai fini del conteggio finale delle assenze: è possibile?

**Comment by trinko** [comment]
No, al momento le assenze in deroga non sono previste.
